src/style/_misc.scss

```diff
@@ -3,6 +3,16 @@ $animation-duration: 250ms;
 $z-index-page-wrapper: 1;
 $z-index-left-nav: 2;
 
-@mixin shadow {
-  box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.1);
+@mixin shadow($elevation: 1) {
+  @if $elevation == 1 {
+    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  }
+
+  @if $elevation == 2 {
+    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+  }
+
+  @if $elevation == 3 {
+    box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
+  }
```

Comment:
New shadows, based on material design. Any existing use of `@include shadow` will default to the first one.